bitnami/keycloak/templates/_helpers.tpl

```diff
@@ -0,0 +1,174 @@
+{{/*
+Create a default fully qualified app name.
+We truncate at 20 chars since the node identifier in WildFly is limited to
+23 characters. This allows for a replica suffix for up to 99 replicas.
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "keycloak.fullname" -}}
```

Comment:
Is there a reason a to why we are using a custom definition instead of using `common.names.fullname`? The only difference that I can see is the length of `trunc` being a smaller number here.

Reply:
Check the comment, it explains why we need to truncate with a lower number of characters.

Reply:
Sorry about that one, you are right. Would you say that this is something worth adding to `common`, with `trunc` being an optional parameter for `common.names.fullname`? Or is it rare enough that it should be solved using a custom `xxxxxx.fullname`?